src/agent/thread_helper.cpp

```diff
@@ -255,9 +255,18 @@ void ThreadHelper::Attach(const std::string &         aNetworkName,
     SuccessOrExit(error = otThreadSetPskc(mInstance, &pskc));
 
     SuccessOrExit(error = otThreadSetEnabled(mInstance, true));
+
+    otbrLog(OTBR_LOG_INFO,
+            "Thread Attach, NetworkName:%s, PanId:0x%04x, ExtPanId:0x%02x%02x%02x%02x%02x%02x%02x%02x, Channel:%u, "
+            "MasterKey: Hidden, PSKc: Hidden",
+            aNetworkName.c_str(), aPanId, extPanId.m8[0], extPanId.m8[1], extPanId.m8[2], extPanId.m8[3],
```

Comment:
Use some method like: `extPanId.ToString()` ?

Reply:
> Use some method like: `extPanId.ToString()` ?

I would recommend
```c++
std::string BytesToHexString(const uint8_t *aBuf, size_t aBufLength);
```

Reply:
Only this place prints the extended panid. There is no need to define a function to convert the panid to string.